### PR TITLE
[SALAD-22777] WebApp: Kitchen demand monitor page 

### PR DESCRIPTION
**Pull request by vitto-moz**
image
image

### CODE REVIEW

**Comment by Maks19** [comment]
I do not see this page on mobile. Is it expected?
image

File: packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorTable/DemandMonitorTable.tsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+              <Text variant="baseXS">Demand</Text>
+            </th>
+            <th className={classes.tableCell}>
+              <Text variant="baseXS">Average Earnings 24/h</Text>
+            </th>
+            <th className={classes.tableCell}>
+              <Text variant="baseXS">Average Running Time 24/h</Text>
+            </th>
+          </tr>
+          {mockedDemandMonitorData.map(({ gpu, hourlyRate, recommendedSpecs, demand, avgEarnings, avgRunningTime }) => (
```
Comment (Maks19) [inline packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorTable/DemandMonitorTable.tsx:103]: Do you plan to sort this data in connection PR?

File: packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorTable/DemandMonitorTable.tsx [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  },
+})
+
+interface Props extends WithStyles<typeof styles> {}
+
+const _DemandMonitorTable: FunctionComponent<Props> = ({ classes }) => {
+  return (
+    <div className={classes.tableWrapper}>
+      <div className={classes.tableContent}>
+        <table className={classes.table}>
```
Comment (Maks19) [inline packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorTable/DemandMonitorTable.tsx:85]: Is it expected to have usual table and not a sortable table?

**Comment by vitto-moz** [comment]
@Maks19 the behavior with the absent `/demand` page in mobile is not expected. I have the page rendered running the app locally in dev mode. 
Your case can be related to the feature flag value, please check this one. Otherwise we have take a more deep look into it 
image

File: packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorTable/DemandMonitorTable.tsx (hunk 2 above)
Comment (vitto-moz) [reply packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorTable/DemandMonitorTable.tsx:85]: could you, please, share more details?
I don't remember we defined the requirements of table sorting

Comment (vitto-moz) [reply packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorTable/DemandMonitorTable.tsx:85]: https://github.com/SaladTechnologies/salad-applications/pull/1219#discussion_r1832706113

File: packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMoniorPage.tsx
Comment (Maks19) [inline packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMoniorPage.tsx:1]: DemandMoniorPage -> DemandMoni**t**orPage

File: packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorTable/DemandMonitorTable.tsx [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+  },
+  gpuWrapper: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    flexDirection: 'column',
+  },
+  gpuName: {
+    marginBottom: '8px',
+  },
```
Comment (Maks19) [inline packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorTable/DemandMonitorTable.tsx:53]:
```suggestion
  gpuWrapper: {
    display: 'flex',
    justifyContent: 'center',
    alignItems: 'center',
    flexDirection: 'column',
    borderTop: `1px ${theme.darkBlue} solid`,
  },
  gpuName: {
    fontWeight: 700,
    marginBottom: '8px',
  },
```

image

File: packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorTable/DemandMonitorTable.tsx [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+            </th>
+            <th className={classes.tableCell}>
+              <Text variant="baseXS">Average Earnings 24/h</Text>
+            </th>
+            <th className={classes.tableCell}>
+              <Text variant="baseXS">Average Running Time 24/h</Text>
+            </th>
+          </tr>
+          {mockedDemandMonitorData.map(({ gpu, hourlyRate, recommendedSpecs, demand, avgEarnings, avgRunningTime }) => (
+            <tr className={classes.tableRow}>
```
Comment (Maks19) [inline packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorTable/DemandMonitorTable.tsx:104]: tableRow is not exist in classes

File: packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorTable/DemandMonitorTable.tsx (hunk 3 above)
Comment (Maks19) [reply packages/web-app/src/modules/demand-monitor-views/DemandMonitorPage/DemandMonitorTable/DemandMonitorTable.tsx:53]: Just a small suggestion to design

**Review by Maks19 — APPROVED** [review]

**Pull request merged**